docs/src/hal/parallel-port.adoc

```diff
@@ -189,48 +190,48 @@ loadrt hal_parport cfg="0x378 0xc000"
 
 Please note that your values will differ.
 The Netmos cards are Plug-N-Play, and might change their settings depending on which slot you put them into,
-so if you like to \'get under the hood' and re-arrange things, be sure to check these values before you start LinuxCNC.
+so if you like to 'get under the hood' and re-arrange things, be sure to check these values before you start LinuxCNC.
 
 == Pins
 
-* 'parport.<p>.pin-<n>-out' (bit) Drives a physical output pin.
-* 'parport.<p>.pin-<n>-in' (bit) Tracks a physical input pin.
-* 'parport.<p>.pin-<n>-in-not' (bit) Tracks a physical input pin, but inverted.
+* `parport.<p>.pin-`__<n>__`-out` (bit) Drives a physical output pin.
+* `parport.<p>.pin-`__<n>__`-in` (bit) Tracks a physical input pin.
+* `parport.<p>.pin-`__<n>__`-in-not` (bit) Tracks a physical input pin, but inverted.
```

Comment:
All the formatting like this is not compatible with our AsciiDoc version (asciidoc-py). It produces wrong and misleading output.

Sorry that I didn't review this PR earlier, but I was a bit unmotivated and a bit tired of always pointng out the same error.

@smoe can you prepare a PR to revert the ``` `.*`__<.*>__`.*` ``` changes?
It is in parallel-port.adoc and rtcomps.adoc - 68 occurences now.

Nevertheless you did a great job on the German translation. I was positively surprised when I picked a page for a German guy from the forum. And you also got some positiv feedback there: https://forum.linuxcnc.org/42-deutsch/48623-linux-cnc-in-oberoesterreich?start=30#268417 😉

Reply:
The only thing that is working is something like `+parport._<p>_.pin-_<n>_-out+` but not compatible with future AsciiDoc versions, so I wouldn't recommend that.

Reply:
Or just leave it as it is to avoid double work for some time in the future when we hopefully have migrated to AsciiDoc?